### PR TITLE
wrapsInnerString: include text following match

### PR DESCRIPTION
**Pull request by josephfrazier**
Fixes https://github.com/OctoLinker/browser-extension/issues/191

### CODE REVIEW

File: lib/insert-link.js [hunk 1]
```diff
@@ -101,9 +101,9 @@ function wrapClosestElement(node, dataAttrObject, matchValue) {
   }
 }
 
-function wrapsInnerString(text, matchValue, dataAttrObject) {
+export function wrapsInnerString(text, matchValue, dataAttrObject) {
```
Comment (stefanbuck) [inline lib/insert-link.js]: I would prefer if we keep this function private. Usually exposing functions just for the purpose of testing is a bad design which ends up most of the times in testing the implementation and not the behaviour. Ideally implementation changes should not affect tests unless the corresponding behaviour is changing as well. Therefor I prefer behaviour over implementation testing.

File: test/insert-link.test.js [hunk 2]
```diff
@@ -127,4 +128,14 @@ describe('helper-replace-keywords', () => {
...
   });
+
+  it('does not remove closing parentheses from commented out require() calls', () => {
+    const text = "// var faker = require('faker')";
+    const options = {
+      'resolver': 'javascriptUniversal',
+      'target': '$1',
+      'path': '/xwartz/robot/blob/3ba4b2628c2252392af58fff0f7231705958ec8d/src/avatar.js',
+    };
+    assert.equal(helper(text, REQUIRE, options).textContent, text);
```
Comment (stefanbuck) [inline test/insert-link.test.js]: I think there is no need to pass any options to the helper in this case.

Comment (josephfrazier) [inline test/insert-link.test.js]: fixed, thanks 👍

**Comment by stefanbuck** [comment]
Awesome 👍 Regarding publishing a new version, let's wait for #190 since this is just a minor issue.

**Comment by josephfrazier** [comment]
That sounds good, thanks!

**Pull request merged**